src/lib/utils/getEndpoint.ts

```diff
@@ -0,0 +1,33 @@
+export type BackendDestination = 'PORTAL_ENDPOINT' | 'SWC_ENDPOINT'
+// These variables indicate which endpoint should be called
+export const PORTAL_ENDPOINT: BackendDestination = 'PORTAL_ENDPOINT'
```

Comment:
why do you have PORTAL_ENDPOINT and SWC_ENDPOINT, isn't that the same thing?  Expected PORTAL (or SWC) and REPO